### PR TITLE
added new table in airtable

### PR DESCRIPTION
**Pull request by ejaz**

### CODE REVIEW

File: air-table/handler.js [hunk 1]
```diff
@@ -109,6 +110,74 @@ exports.updatePreferences = async event => {
...
+      )
+    })
+  } else {
+    const { shopify_customer_id, shopify_order_id, order_products } = body
+    if (!shopify_customer_id || !shopify_order_id || !order_products || customer_email)
+      return generateResponse({ message: 'please send all fields' }, 400, event)
+    row = new Promise((res, rej) => {
+      ordersTable('orderHistory').create([{ fields: { ...body.fields } }], function done(err, record) {
+        if (err) {
+          console.log('Error while fetching row', err)
```
Comment (PygmalionPolymorph) [inline air-table/handler.js]: This error message is wrong

File: air-table/handler.js [hunk 2]
```diff
@@ -109,6 +110,74 @@ exports.updatePreferences = async event => {
...
+          if (err) {
+            return rej(err)
+          }
+          return res({ message: 'Row has been updated' })
+        }
+      )
+    })
+  } else {
+    const { shopify_customer_id, shopify_order_id, order_products } = body
+    if (!shopify_customer_id || !shopify_order_id || !order_products || customer_email)
```
Comment (PygmalionPolymorph) [inline air-table/handler.js]: Is the `|| customer_email` correct or should this be `!customer_email` ? I'm not sure how the semantics are..

File: air-table/handler.js (hunk 1 above)
Comment (ejaz) [reply air-table/handler.js]: changed message to "unable to fetch preferences"

File: air-table/handler.js (hunk 2 above)
Comment (ejaz) [reply air-table/handler.js]: this is correct logic

**Pull request merged**